application/src/main/java/org/exoplatform/chat/portlet/notification/templates/index.gtmpl

```diff
@@ -10,6 +10,7 @@
   data-chat-interval-notif="<%=chatIntervalNotif%>"
   data-chat-interval-status="<%=chatIntervalStatus%>"
   data-short-space-name="<%=shortSpaceName%>"
+  data-db-name="<%=dbName%>"
```

Comment:
ok, I did not see that the hasUserWithToken has been updated as well to use the dbName. Well done.